corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -36,6 +36,8 @@
 @register.simple_tag
 def render_form_xml(form):
     xml = form.get_xml()
+    if isinstance(xml, unicode):
+        xml.encode('utf-8', errors='replace')
```

Comment:
Excellent point! We should take that out sometime, and the other instance in [XFormInstance._xml_string_to_element()](https://github.com/dimagi/commcare-hq/blob/df78fea8ada7124042e3bb9cb6ce166b8212c298/corehq/ex-submodules/couchforms/models.py#L275-L275).